### PR TITLE
feat: refine user flows & styling

### PR DESCRIPTION
**Pull request by Aidasvenc**
# What I did
Addressed the following issues and user feedback mentioned in #144:

- Make section tabs easier to press on.
- Improve the password: make an option to hide the password with "****" in registration and finish all checks (some of the checks like password strength are not functional I don't remember which ones).
- After adding a description, the fact that there is still a button “Add a description” made users feel weird. Change the button to “Modify description” or remove it after the user has put their description.
-  On login the button "Continue with google" should have a capital G.
- In registration, “Please check your data” is not a good error message, it doesn’t tell the user what’s wrong.
- Make the screen to view all the announcements in a list.

Also, did some styling changes in order to have a more consistent design language and to focus on visibility of main components in the application.

# How I did it
- Changed layouts of different components and screens.
- Changed styling of different components and screens

# How to verify it
- Run the app and explore it.

# Demos
WhatsApp Image 2024-05-06 at 22 26 23_19fa951f
WhatsApp Image 2024-05-06 at 22 26 23_ee5ddf31
WhatsApp Image 2024-05-06 at 22 26 24_d4da9f1a
WhatsApp Image 2024-05-06 at 22 26 24_442e5819

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [ ] work correctly on Android
- [x] are fully tested

### CODE REVIEW

File: screens/Contacts/ContactGraph/styles.ts [hunk 1]
```diff
@@ -20,8 +20,9 @@ export const styles = StyleSheet.create({
     borderColor: lightGray,
     borderRadius: 40,
     borderWidth: 1,
-    height: 60,
-    margin: 10,
-    padding: 20,
+    fontFamily: "JetBrainsMono_400Regular",
```
Comment (AlbertoCentonze) [inline screens/Contacts/ContactGraph/styles.ts:23]: Don't we have a constant for the font in the global style?

File: screens/Contacts/ContactList/styles.ts [hunk 2]
```diff
@@ -43,8 +43,9 @@ export const styles = StyleSheet.create({
     borderColor: lightGray,
     borderRadius: 40,
     borderWidth: 1,
-    height: 60,
-    margin: 10,
-    padding: 20,
+    fontFamily: "JetBrainsMono_400Regular",
```
Comment (AlbertoCentonze) [inline screens/Contacts/ContactList/styles.ts:46]: Same as above

**Review by AlbertoCentonze — CHANGES_REQUESTED** [review]
Hi, good job, should have probably split the PR into multiple ones but issok.
I've mentioned it a couple of times and stopped to avoid flooding you with comments but I think we have constants for the font names, and if we don't we should probably create them.

Also I start noticing some spacing issues in the code, don't forget to use prettier to make it look prettier ;)

File: screens/Contacts/ContactGraph/styles.ts (hunk 1 above)
Comment (Aidasvenc) [reply screens/Contacts/ContactGraph/styles.ts:23]: True

**Comment by Aidasvenc** [comment]
> Hi, good job, should have probably split the PR into multiple ones but issok. I've mentioned it a couple of times and stopped to avoid flooding you with comments but I think we have constants for the font names, and if we don't we should probably create them.
> 
> Also I start noticing some spacing issues in the code, don't forget to use prettier to make it look prettier ;)

Honestly I hate prettier, it makes the code unreadable in some cases:

image

**Comment by AlbertoCentonze** [comment]
> Honestly I hate prettier, it makes the code unreadable and spaces the code into many lines in some cases:

I get you, we should probably take some time to create a good .prettierrc. Clearly out of scope for this sprint.

Thanks for the fixes

**Review by AlbertoCentonze — APPROVED** [review]

**Pull request merged**